roles/include_components/tasks/track_repo_commit_component.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Assert ic_commit_url
+  assert:
+    that:
+      # Assert that ic_commit_url is a valid URL
+      # of the sort https://github.com/organisation/reponame/commit/commit_hash
+      - ic_commit_url is match('^https://github\\.com/[^/]+/[^/]+/commit/[0-9a-f]{40}$')
```

Comment:
This ensures you have to provide the full SHA as a component, if that is the case the update the docs stating you need the full SHA